### PR TITLE
copr: Add vectorized crc32

### PR DESCRIPTION
**Pull request by gauss1314**
Signed-off-by: gauss <gauss1314@gmail.com>

PCP #5751 

###  What have you changed?

Add vectorized crc32

###  What is the type of the changes?

- Improvement (a change which is an improvement to an existing feature)

###  How is the PR tested?

- Unit test
- Integration test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No

###  Does this PR affect `tidb-ansible`?

No

### CODE REVIEW

File: components/tidb_query/src/rpn_expr/impl_math.rs [hunk 1]
```diff
@@ -1,5 +1,7 @@
 use tidb_query_codegen::rpn_fn;
 
+use crc::{crc32, Hasher32};
```
Comment (breeswish) [inline components/tidb_query/src/rpn_expr/impl_math.rs]: I would prefer to put it in `fn crc32`.

**Comment by gauss1314** [comment]
@sre-bot /run-integration-copr-test copr-test=pr/66

**Comment by gauss1314** [comment]
@sre-bot /run-integration-copr-test copr-test=pr/66

**Pull request merged**